index.yaml

```diff
@@ -46,6 +46,7 @@ defs:
   PT: ['pt.yaml']
   RO: ['ro.yaml']
   SE: ['se.yaml']
+  TN: ['tn.ymal']
```

Comment:
Woops, sorry. I overlooked this.
Could you modify a typo? ( 'tn.yaml' is correct)